src/sentry/static/sentry/app/components/smartSearchBar/index.jsx

```diff
@@ -266,7 +269,13 @@ class SmartSearchBar extends React.Component {
   };
 
   doSearch = async () => {
-    const {onSearch, onSavedRecentSearch, api, orgId, savedSearchType} = this.props;
+    const {
+      onSearch,
+      onSavedRecentSearch,
+      api,
+      organization,
+      savedSearchType,
```

Comment:
Where does this get set to a different value for events vs. issues?

Reply:
@markstory it comes from props via https://github.com/getsentry/sentry/pull/14420/files#diff-710110490bc7faf582c4f8c1b6eaafcbR87

Reply:
It comes from props via https://github.com/getsentry/sentry/pull/14420/files#diff-710110490bc7faf582c4f8c1b6eaafcbR87